sigar_linux.go

```diff
@@ -137,6 +137,103 @@ func (self *CpuList) Get() error {
 
 	return err
 }
+func (self *NetProtoV6Stats) Get() error {
+	return readFile(Procd+"/net/snmp6", func(line string) bool {
+		fields := strings.Fields(line)
+		if len(fields) != 2 {
```

Comment:
nit: maybe include a comment saying the expected records have key and value, and other lines are ignored?